File: src/server/gkill/api/req_res/update_mi_re_kyou_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateMiReKyouResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedMiReKyou *reps.MiReKyou `json:"updated_mirekyou"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}

File: src/client/classes/api/req_res/update-mi-re-kyou-response.ts
'use strict'

import { MiReKyou } from '@/classes/datas/mi-re-kyou'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateMiReKyouResponse extends GkillAPIResponse {

    updated_mirekyou: MiReKyou

    updated_kyou: Kyou | null

    constructor() {
        super()
        this.updated_mirekyou = new MiReKyou()
        this.updated_kyou = null
    }

}
